Sources/Components/SelectionBoxes/Selectionbox.swift

```diff
@@ -0,0 +1,199 @@
+            for item in items {
+                item.imageView.highlightedImage = selectedImage
+            }
+        }
+    }
+
+    public var selectedAnimationImages: [UIImage]? {
+        didSet {
+            guard let items = stack.arrangedSubviews as? [SelectionboxItem] else { return }
+            for item in items {
+                item.imageView.highlightedAnimationImages = selectedAnimationImages
+                if let selected = selectedAnimationImages {
+                    item.imageView.selectedDuration = Double(selected.count) / AnimatedImageView.framesPerSecond
+                }
+            }
+        }
+    }
+
+    public var title: String? {
+        get { return titleLabel.text }
+        set { titleLabel.text = newValue }
+    }
+
+    // MARK: Private properties
+
+    private var highlightedItem: SelectionboxItem?
+
+    private let titleLabel: UILabel = {
+        let label = Label(style: .title3)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let stack: UIStackView = {
+        let stack = UIStackView(frame: .zero)
+        stack.axis = .vertical
+        stack.distribution = .fillEqually
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        return stack
+    }()
+
+    // MARK: Implementation
+
+    public init(strings: [String]) {
+        super.init(frame: .zero)
+        setupBoxes(with: strings)
+    }
+
+    public override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        guard let touch = touches.first else { return }
+        let location = touch.location(in: self)
+        guard let item = hitTest(location, with: event) as? SelectionboxItem else { return }
+        highlightedItem = item
+    }
+
+    public override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+        guard let touch = touches.first else { return }
+        let location = touch.location(in: self)
+        guard let item = hitTest(location, with: event) as? SelectionboxItem else { return }
+        guard item === highlightedItem else { return }
+        handleSelecting(item)
+    }
+
+    fileprivate func handleSelecting(_ item: SelectionboxItem) {
+        fatalError("This class should not be used directly")
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension Selectionbox {
+    
+    // MARK: Private methods
+    
+    private func setupBoxes(with strings: [String]) {
+        for (i, string) in strings.enumerated() {
+            let item = SelectionboxItem(index: i)
+            item.label.text = string
+            stack.addArrangedSubview(item)
+        }
+
+        addSubview(titleLabel)
+        addSubview(stack)
+
+        NSLayoutConstraint.activate([
+            titleLabel.leftAnchor.constraint(equalTo: leftAnchor, constant: Spacing.mediumLargeSpacing),
```

Comment:
Since we've implemented this as an extension of CGFloat as well you can drop the `Spacing` and just do `.mediumLargeSpacing`.